src/appengine/libs/issue_management/github.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2022 Google LLC
```

Comment:
Might be easier to name this module something else since the pygithub package's module is also called "github"

Reply:
It's fine for now. We shouldn't ever use the PyGitHub package directly.

Reply:
Thinking more, this module is pretty specific to OSS-Fuzz, so let's name this "oss_fuzz_github.py"

Reply:
Renamed to in `oss_fuzz_github.py` [6f1ed6b](https://github.com/google/clusterfuzz/pull/2578/commits/6f1ed6bc988410f91e64ea5335ae5d968d1fb778)